src/lib/divider/divider.ts

```diff
@@ -0,0 +1,55 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {
+  ChangeDetectionStrategy, Component,
+  Directive,
+  Input, ViewEncapsulation,
+} from '@angular/core';
+import {coerceBooleanProperty} from '@angular/cdk/coercion';
+
+@Component({
+  moduleId: module.id,
+  selector: 'mat-divider',
+  host: {
+    'role': 'separator',
+    '[attr.aria-orientation]': 'vertical ? "vertical" : "horizontal"',
+    '[class.mat-divider-vertical]': 'vertical',
+    '[class.mat-divider-inset]': 'inset',
+  },
+  template: '<ng-content></ng-content>',
```

Comment:
I am not quite sure I see the value in being able to project content into the `mat-divider` element that is created.  What would be the use case for this?

Reply:
Changed